README.md

```diff
@@ -16,12 +16,79 @@ goose
 <a href="#open-source">Open Source</a>
 </p>
```

Comment:
**General feedback from Brad:**

> 1. I'd suggest we avoid direct comparisons to other tools, like the grid or some of the content in the what sets it apart section. I'd probably send this a bit more toward a framing like "Goose is not looking to do code line completions or pair programming" or similar, and then say what it is trying to do. And leave the vs copilot, vs aider, vs cursor implied? And we can put lines about why we think that is a valuable approach/principle compared to other approaches
> 2. Maybe the examples can grow into a "what can you do with goose" section? where we have prompts like you have here, and then maybe talk through why we prompt it like that (effectively tips again) and how we have seen it solve that thing
    a. we can probably manufacture more open examples btw, these are real which is great but some are block specific

Reply:
My thoughts on the grid is it's not currently very compelling (for goose) but I like the idea. 

@baxen I'd be curious to hear more about what exactly you don't like about the grid. Is it that you don't find it compelling as it currently is, or you think that a direct comparison isn't something users would be interested in? Something else?